src/sass/_common/common.scss

```diff
@@ -674,6 +674,10 @@ ul.bullet.gr-row > li {
     display: inline-block;
 }
 
+.change-account-btn {
+    margin-left: 0 !important;
```

Comment:
This is a workaround, that lets us leave other buttons' layout untouched. "!important" is widespread on binary, Carol approved it once

Reply:
I mean approved using !important

Reply:
you can avoid important by specifying the selector more, e.g a.change-account-btn.button

Reply:
I agree, better to be more specific and remove important. just because we used it previously doesn't mean we should follow that bad practice.

Reply:
changed to more specific selector

Reply:
We should use it for unavoidable cases only @Daniil-Deriv